test/Extism.Sdk/BasicTests.cs

```diff
@@ -178,6 +180,51 @@ public void HostFunctionsWithMemory()
         Encoding.UTF8.GetString(response).ShouldBe("HELLO FRODO!");
     }
 
+    [Fact]
+    public void FileLog()
+    {
+        var tempFile = Path.GetTempFileName();
+        Plugin.ConfigureFileLogging(tempFile, LogLevel.Warn);
+        using (var plugin = Helpers.LoadPlugin("log.wasm"))
+        {
+            plugin.Call("run_test", Array.Empty<byte>());
+        }
+
+        // HACK: tempFile gets locked by the Extism runtime 
+        var tempFile2 = Path.GetTempFileName();
+        File.Copy(tempFile, tempFile2, true);
+
+        var content = File.ReadAllText(tempFile2);
+        content.ShouldContain("warn");
+        content.ShouldContain("error");
+        content.ShouldNotContain("info");
+        content.ShouldNotContain("debug");
+        content.ShouldNotContain("trace");
+    }
+
+
+    // [Fact]
+    // Interferes with FileLog
+    internal void CustomLog()
+    {
+        var builder = new StringBuilder();
+
+        Plugin.ConfigureCustomLogging(LogLevel.Warn);
+        using (var plugin = Helpers.LoadPlugin("log.wasm"))
+        {
+            plugin.Call("run_test", Array.Empty<byte>());
+        }
+
+        Plugin.DrainCustomLogs(line => builder.AppendLine(line));
+
+        var content = builder.ToString();
+        content.ShouldContain("warn");
+        content.ShouldContain("error");
+        content.ShouldNotContain("info");
+        content.ShouldNotContain("debug");
+        content.ShouldNotContain("trace");
+    }
```

Comment:
@zshipko I had to comment this test out because it was interfering with the `FileLog` test (they were eating each others logs). Is there any way to disable Custom/File logging?

Reply:
Not sure how hard it is, but I think it would be good if they automatically unsubscribe each other

Reply:
I spent some time looking at this today and I think we could use https://docs.rs/tracing-subscriber/latest/tracing_subscriber/reload/index.html but it isn't super straight-forward. I will see if I can come up with a minimally invasive way to use that but haven't had much luck so far.